biogtr/inference/track.py

```diff
@@ -0,0 +1,123 @@
+"""Script to run inference and get out tracks."""
+
+from biogtr.config import Config
+from biogtr.models.gtr_runner import GTRRunner
+from biogtr.datasets.tracking_dataset import TrackingDataset
+from omegaconf import DictConfig
+from pprint import pprint
+from pathlib import Path
+
+import os
+import hydra
+import pandas as pd
+import pytorch_lightning as pl
+import torch
+
+device = "cuda" if torch.cuda.is_available() else "cpu"
+
+torch.set_default_device(device)
+
+
+def inference(
```

Comment:
This is fine for now, but we should clean up a bit once we work on the tracker side of things.